bp_common/test/mem/bootrom.dump

```diff
@@ -0,0 +1,56 @@
+
+/mnt/users/ssd0/no_backup/petrisko/pre-alpha-release/bp_top/syn/results/vcs/bp_tethered.e_bp_unicore_cfg.none.sim.bp_tests.hello_world/bootrom.riscv:     file format elf64-littleriscv
```

Comment:
Is it possible / do you want to remove this full path since it is specific to you?

Reply:
Good catch, will remove